app/javascript/components/Study.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const descriptionCharLimit = 750
+
+export function formatDescription(rawDescription) {
+
+  let textDescription = stripTags(rawDescription)
+  let suffixString = ''
+  if (textDescription.length > descriptionCharLimit) {
+    suffixString = '... (continued)'
+  }
+  return `${textDescription.substring(0, descriptionCharLimit)}${suffixString}`
+}
+
+function stripTags(rawString) {
+  let tempDiv = document.createElement('div');
+  // Set the HTML content with the providen
+  tempDiv.innerHTML = rawString;
+  // Retrieve the text property of the element
+  return tempDiv.textContent || '';
+}
+
+export default function Study(props) {
```

Comment:
Please add a brief docstring to exported functions, for developer convenience.  E.g.:

```suggestion
/**
* Component for study search result.  Reports study name, accession, and brief description.
*/
export default function Study(props) {
```

(I consider `@param` docs for `props` extraneous, but I wouldn't object to including them.)